guides/doc-Configuring_virt-who_VM_Subscriptions/master.adoc

```diff
@@ -0,0 +1,170 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: virt-who-vm-subs
+:virt-who-vm-subs:
+:ProductName: {ProjectServer}
+
+= {ConfiguringVMSubscriptionsDocTitle}
+
+ifdef::foreman-deb[]
+You cannot use the virt-who plugin because it requires the Katello plugin, which is not supported on Debian or Ubuntu.
+endif::[]
+ifdef::foreman-el[]
+You cannot use the virt-who plugin without the Katello plugin.
+endif::[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,satellite,orcharhino[]
+include::common/assembly_overview-of-vm-subscriptions.adoc[leveloffset=+1]
+
+:!virt-who-vm-subs:
```

Comment:
You don't have to cancel this attribute. It can be defined through the whole guide.

Reply:
There would be a problem with `:context:`, but not with these custom ones.

Reply:
I will give it a try!